spec/index.bs

```diff
@@ -2210,20 +2180,125 @@ enum GPUTextureAspect {
+                                <dl class="switch">
+                                    : {{GPUTextureViewDimension/"1d"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"1d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `1`.
+
+                                    : {{GPUTextureViewDimension/"2d"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `1`.
+
+                                    : {{GPUTextureViewDimension/"2d-array"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
+
+                                    : {{GPUTextureViewDimension/"cube"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `6`.
+                                    :: |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/width=] is
+                                        |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/height=].
+
+                                    : {{GPUTextureViewDimension/"cube-array"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is a multiple of `6`.
+                                    :: |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/width=] is
+                                        |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/height=].
+
+                                    : {{GPUTextureViewDimension/"3d"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"3d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `1`.
+                                </dl>
+                        </div>
+
+                        Then:
+                            1. Generate a {{GPUValidationError}} in the current scope with appropriate error message.
+                            1. Create a new [=invalid=] {{GPUTextureView}} and return the result.
+
+                    1. Let |view| be a new {{GPUTextureView}} object.
+                    1. Set |view|.{{GPUTextureView/[[texture]]}} to |this|.
+                    1. Set |view|.{{GPUTextureView/[[descriptor]]}} to |descriptor|.
+                    1. Return |view|.
+                </div>
+        </div>
 </dl>
 
+Issue: Define what it means to be a <dfn dfn>compatible texture format</dfn> with another texture format.
+
+<div algorithm>
+    When <dfn abstract-op>resolving GPUTextureViewDescriptor defaults</dfn> for {{GPUTextureViewDescriptor}}
```

Comment:
nice way of defining the defaults!

Reply:
agreed, I wanted to do a formalization of defaulting and this looks good 👍